cubic-server/Player.cpp

```diff
@@ -575,19 +589,31 @@ void Player::_onSetPlayerPosition(const std::shared_ptr<protocol::SetPlayerPosit
 {
     LDEBUG("Got a Set Player Position (", pck->x, ", ", pck->feet_y, ", ", pck->z, ")");
     // TODO: Validate the position
+    if (pck->on_ground && _isFlying)
+        _isFlying = false;
+    if (!pck->on_ground && !_isJumping && !_isFlying && ((pck->feet_y + 64) - (_pos.y + 64) > 0)) {
```

Comment:
Here there is a magic value, please name it so we can know afterwards what it is.

Reply:
`world_storage::CHUNK_HEIGHT_MIN` in [chunkcolumn.cpp](/cubic-server/world_storage/ChunkColumn.cpp)
```suggestion
    if (!pck->on_ground && !_isJumping && !_isFlying && ((pck->feet_y + (-world_storage::CHUNK_HEIGHT_MIN)) - (_pos.y + (- world_storage::CHUNK_HEIGHT_MIN)) > 0)) {
```